CDCgov/NEDSS-Modernization
# Nifi auto start ingestion

## Description

Updates the Patient initial ingestion process group to default to `ON`. Adds a log message every 5 minutes that will show the current progress of `Elasticsearch person documents / Database MPRs`.

NiFi has been updated to have a persistent volume. Due to this persistent storage, the ingestion processes should not restart on a crash. The foundation team has requested the initial ingestion be set to `ON` by default and that there be some log message showing progress.

## Tickets

N/A

## Steps to verify

1. Rebuild NiFi container (remove old volumes)
1. Log into [NiFi ](https://localhost:8443/nifi/)
1. Verify all Patient search ingestion processes are running 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/58626341-0b20-488c-b9f7-b805cd3f0b56)
1. View NiFi log file
1. Verify progress log exists
```
2023-10-03 12:53:04 2023-10-03 16:53:03,855 INFO [Timer-Driven Process Thread-6] o.a.nifi.processors.standard.LogMessage LogMessage[id=f222491b-018a-1000-4dce-eccda6d27072] Elasticsearch ingestion status: 31 / 31 Persons (Status Check Every 5 Minutes)
```
